src/server/saved_objects/client/lib/parse_es_doc.js

```diff
@@ -0,0 +1,16 @@
+import { get } from 'lodash';
+
+export function parseEsDoc(doc, overrides = {}) {
```

Comment:
The overrides are also for things like update/create/bulkCreate where ES doesn't return the source, type, etc.